lib/ui/src/components/layout/container.tsx

```diff
@@ -120,7 +120,7 @@ export const Main: FunctionComponent<{ isFullscreen: boolean; position: CSSPrope
   position = undefined,
   ...props
 }) => (
-  <Pane style={position} top {...props}>
+  <Pane style={position} top {...props} role="main">
```

Comment:
This is totally correct so I approved. But as a suggestion, a better way would be to render a ˋ<main>` élément

Via styled component « as »
```suggestion
  <Pane style={position} top {...props} as="main">
```

The same can be applied to the sidebar to render a `nav` element

Reply:
Great suggestions! I'm hoping to use this one as an example, but then I'd like this approach to be spread so that most if not all major UI elements have landmarks set like this. I'll make the `as` update after testing to confirm axe picks it up correctly.

Reply:
@jsomsanith we're using emotion not styled components ... not sure that makes a difference? 🤷‍♂️

Reply:
Thanks for the precision. Emotion has the « as » props as well 🙂

Reply:
This took me a bit longer to get back to than expected, but from what I can tell, `as` can't be used for this type of element:


```
Type '{ children: Element; as: string; style: CSSProperties; top: true; }' is not assignable to type 'IntrinsicAttributes & ClassAttributes<HTMLDivElement> & HTMLAttributes<HTMLDivElement> & Pick<...> & { ...; } & { ...; }'.
  Property 'as' does not exist on type 'IntrinsicAttributes & ClassAttributes<HTMLDivElement> & HTMLAttributes<HTMLDivElement> & Pick<...> & { ...; } & { ...; }'.ts(2322)
```